browser/brave_browser_main_extra_parts.cc

```diff
@@ -13,10 +12,6 @@ BraveBrowserMainExtraParts::BraveBrowserMainExtraParts() {
 BraveBrowserMainExtraParts::~BraveBrowserMainExtraParts() {
 }
 
-void BraveBrowserMainExtraParts::PreCreateThreads() {
-  brave::InitializeResourceBundle();
```

Comment:
This was removed because it wasn't the right place to initialize this. It was causing a crash at startup for service main refactoring. See here:
https://github.com/brave/brave-core/pull/1117/commits/ae71f9bae04496631e726e7ad16c75efffea6f22